src/benchmark/Akka.Benchmarks/Actor/ActorPathBenchmarks.cs

```diff
@@ -20,18 +21,25 @@ public class ActorPathBenchmarks
         private Address _sysAdr = new Address("akka.tcp", "system", "127.0.0.1", 1337);
         private Address _otherAdr = new Address("akka.tcp", "system", "127.0.0.1", 1338);
 
+        private string _actorPathStr;
+        
+        [Params(1, 100000, int.MaxValue)]
+        public int Uid { get; set; }
+
         [GlobalSetup]
         public void Setup()
         {
             x = new RootActorPath(_sysAdr, "user");
             y = new RootActorPath(_sysAdr, "system");
-            _childPath = x / "parent" / "child";
+            var parentPath = x / "parent";
+            _childPath = new ChildActorPath(parentPath, "child", Uid);
+            _actorPathStr = _childPath.ToSerializationFormat();
```

Comment:
Have the string we parse be produced by the absolute path of the `ChildActorPath` we use for the other tests, including the UID.